src/v/coproc/partition_manager.cc

```diff
@@ -77,6 +77,7 @@ ss::future<> partition_manager::remove(const model::ntp& ntp) {
           ntp));
     }
 
+    vlog(coproclog.info, "Removing materialized log: {}", ntp);
```

Comment:
this seems like unrelated changes.

Reply:
Yeah i wasn't sure where to put it, but it makes debugging much easier, should i make a separate commit?